reference/constraints/IsTrue.rst

```diff
@@ -113,7 +112,9 @@ Then you can constrain this method with ``IsTrue``.
             }
         }
 
-If the ``isTokenValid()`` returns false, the validation will fail.
+If the ``isTokenValid()`` returns false, the validation will fail. When applied
+to methods, this constraint requires method names to start with ``is`` or
```

Comment:
This does not look right to me. Is this based on some bug report?

Reply:
The original doc said that this works with *"a "getter" method"*  But what's that? Only getters? Because I know that it works for issers too ... so, getters + issers only? Or does this work with ANY kind of method regardless of its name? Thanks.

Reply:
This works with an method that returns a boolean value.

Reply:
Thanks for clarifying this. I've updated the description.